src/ducks/pin/PinGuard.jsx

```diff
@@ -0,0 +1,198 @@
+/* global Fingerprint */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import range from 'lodash/range'
+import styles from './styles.styl'
+
+const Round = props => {
+  return <button {...props} className={styles.Round} />
+}
+
+class FingerprintAuthButton extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {}
+    this.handleClick = this.handleClick.bind(this)
```

Comment:
tous les `.bind(this)` ne sont pas obligatoire.

Reply:
C'est babel. Voir la guideline ici pour ce sujet : https://github.com/cozy/cozy-guidelines#binding-event-handlers